swift/Tests/SwiftNoosphereTests/NoosphereTests.swift

```diff
@@ -407,4 +407,189 @@ final class NoosphereTests: XCTestCase {
         ns_sphere_fs_free(sphere_fs)
         ns_free(noosphere)
     }
+    
+    func testSettingAndGettingAPetname() throws {
+        let noosphere = ns_initialize("/tmp/foo", "/tmp/bar", nil, nil)
+
+        ns_key_create(noosphere, "bob", nil)
+
+        let sphere_receipt = ns_sphere_create(noosphere, "bob", nil)
+
+        let sphere_identity_ptr = ns_sphere_receipt_identity(sphere_receipt, nil)
+        let sphere = ns_sphere_fs_open(noosphere, sphere_identity_ptr, nil)
+        
+        ns_string_free(sphere_identity_ptr)
+        ns_sphere_receipt_free(sphere_receipt)
+        
+        ns_sphere_petname_set(noosphere, sphere, "alice", "did:key:alice", nil)
```

Comment:
Future Thoughts: Wonder if we can bake the noosphere context into the sphere to simplify API in the future by removing the `noosphere` arg when a `sphere` has been provided (and "opened" via a context):

```c
ns_sphere_t sphere = ns_sphere_fs_open(noosphere, "did:key:bob",  NULL);
ns_sphere_petname_set(sphere, "alice", "did:key:alice", NULL);
```

Reply:
It's possible, file an issue?

Reply:
@jsantell filed #272